File: 1598-crawler-log-folder/1598-crawler-log-folder.js
/**
 * @param {string[]} logs
 * @return {number}
 */
var minOperations = function(logs) {
    
    let output = 0
    
    for (let i = 0; i < logs.length; i++) {
        let curr = logs[i]
        if (curr === "./") {
            output = output
        } else if (curr === '../') {
            if(output > 0) output -= 1;
        } else {
            output++;
        }
    }
    
    return output
    
};

File: 1598-crawler-log-folder/1598-crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        output = 0
        for log in logs:
            if (log == "./"):
                output = output
            elif (log == '../'):
                if (output > 0): 
                    output = output - 1 
            else:
                output = output + 1
                
        return output
        